store/map.js

```diff
@@ -42,54 +42,67 @@ export const mutations = {
 };
 
 export const actions = {
-  createMap(ctx, {map}) {
+  createMap(ctx, { map }) {
     const { commit } = ctx;
-    commit(types.CREATE_MAP, {map});
+    commit(types.CREATE_MAP, { map });
   },
 
-  async getMap(ctx, {id}){
+  async getMap(ctx, { id }) {
```

Comment:
just do `({ commit }, { id })` instead of this `ctx, commit = ctx` stuff you're doing.